types/downloadjs/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for downloadjs 1.4
+// Project: http://danml.com/download.html
+// Definitions by: cwmoo740 <https://github.com/cwmoo740>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare namespace download {
+    type Data = string | File | Blob;
+    interface Stringable {
+        toString(): string;
+    }
+}
+declare function download(data: download.Data, filename?: string, mimeType?: string): void;
+declare function download<T extends download.Stringable>(data: T, filename?: string, mimeType?: string): void;
```

Comment:
The library provides this example:

```js
var str= "hello world",	arr= new Uint8Array(str.length);
str.split("").forEach(function(a,b){
  arr[b]=a.charCodeAt();
});

download( arr, "textUInt8Array.txt", "text/plain" );
```
I'm not sure why anyone would want to do that, but given that the library calls toString() on whatever is passed as the first argument, I thought I would add the type for that. 

I'm happy removing it too.